recipes-bsp/opensbi/opensbi_%.bbappend

```diff
@@ -8,3 +8,6 @@ SRC_URI:append:nezha = " \
 "
 
 INSANE_SKIP:${PN}:nezha += "ldflags"
+
+# Support fdt drivers for AE350
+SRCREV:ae350-ax45mp = "22f38ee6c658a660083aa45c4ec6c72f66a17260"
```

Comment:
can we bump it in OE-Core along side too? so we can remove this once in core ?

Reply:
OpenSBI v1.2 is not yet available.
:(